src/harness/unittests/typingsInstaller.ts

```diff
@@ -1051,7 +1051,7 @@ namespace ts.projectSystem {
             session.executeCommand(changeRequest);
             host.checkTimeoutQueueLengthAndRun(2); // This enqueues the updategraph and refresh inferred projects
             const version2 = proj.getCachedUnresolvedImportsPerFile_TestOnly().getVersion();
-            assert.equal(version1, version2, "set of unresolved imports should not change");
+            assert.notEqual(version1, version2, "set of unresolved imports should change");

```

Comment:
Ah, the beauty of "Why didn't we have tests for this?". "We do, they just expected the wrong behavior" :-)